t/app/controller/dashboard.t

```diff
@@ -122,7 +122,7 @@ FixMyStreet::override_config {
         $mech->get_ok("/dashboard?ward=$alt_area_id");
         $mech->content_contains('<h1>Trowbridge</h1>');
 
-        $counciluser->update({area_id => undef});
+        $counciluser->update({area_ids => undef});
     };
```

Comment:
We probably should have a test that checks assigning multiple area ids just for completeness